elastic/integrations
# [ IIS ] - Replace deprecated control filter with new control

- Enhancement

## What does this PR do?

Replaces the older deprecated control filter with new control panel. The old filter doesn't respect the selected time range in Kibana to load the data.

The Kibana version has been updated to `8.3.0` as the new control panel is introduced in 8.3.0

## Checklist



## How to test this PR locally

Verify the control filter is replaced with the new one.
Check the dashboards loading data.

## Related issues


- Closes #4302
